app/models/section.rb

```diff
@@ -6,6 +6,8 @@ class Section < ApplicationRecord
     has_many :student_sections
     belongs_to :course
 
-    validates_presence_of :name, :location, :building, :room, :start_time, :end_time, :days
+    # validates_presence_of :name, :location, :building, :room, :start_time, :end_time, :days
+    validates_presence_of :name, :location
```

Comment:
Is this because the ScottyLabs API doesn't have some of these values? What are we thinking of doing if they are null?

Reply:
Yeah ScottyLabs is just returning nil values I feel like we have to just ignore them (?)